R/f7Preloader.R

```diff
@@ -63,33 +39,79 @@
 #'       )
 #'     ),
 #'     server = function(input, output, session) {
+#'       long_task <- eventReactive(input$obs, {
+#'         Sys.sleep(4)
+#'         input$obs
+#'       })
+#'
+#'       observeEvent(input$obs, {
+#'         showF7Preloader(color = "red", type = "dialog", id = "loader")
```

Comment:
@hypebright this is similar to {waiter} and it might feel more natural to be able to chain functions. However, I don't want to depends on R6. Maybe S4...